examples/python/crashlog.py

```diff
@@ -221,7 +221,7 @@ def dump(self, prefix):
 
     class DarwinImage(symbolication.Image):
         """Class that represents a binary images in a darwin crash log"""
-        dsymForUUIDBinary = os.path.expanduser('~rc/bin/dsymForUUID')
+        dsymForUUIDBinary = '/usr/local/bin/dsymForUUID'
         if not os.path.exists(dsymForUUIDBinary):
             try:
                 dsymForUUIDBinary = subprocess.check_output('which dsymForUUID',
```

Comment:
Looks good, thanks!  Did you want to add a `.rstrip()` to the output of `subprocess.check_output()` in case /usr/local/bin/dsymForUUID doesn't exist but it's in a different location?